create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusS3Client.java

```diff
@@ -0,0 +1,98 @@
+package no.sikt.nva.scopus;
+
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ScopusS3Client {
+
+    private static final Logger logger = LoggerFactory.getLogger(ScopusS3Client.class);
+    public static final String COULD_NOT_GET_FILE = "Could not get file: {}";
+    public static final String COULD_NOT_LIST_FILES_FOR = "Could not list files for: {}";
+
+    private String bucketName;
+    private AmazonS3 amazonS3Client;
+    protected static final String AWS_REGION = "AWS_REGION";
+    protected static final String BUCKET_NAME = "BUCKET_NAME";
+    public static final String CANNOT_CONNECT_TO_S3 = "Cannot connect to S3";
+
+    @JacocoGenerated
+    public ScopusS3Client() {
+        initS3Client(new Environment());
+    }
+
+    @JacocoGenerated
+    public ScopusS3Client(Environment environment) {
+        initS3Client(environment);
+    }
+
+    /**
+     * Constructor for use in test to inject.
+     *
+     * @param amazonS3Client aws S3 client
+     * @param bucketName     name til s3 bucket
+     */
+    public ScopusS3Client(AmazonS3 amazonS3Client, String bucketName) {
+        this.amazonS3Client = amazonS3Client;
+        this.bucketName = bucketName;
+    }
+
+    @JacocoGenerated
+    private void initS3Client(Environment environment) {
+        try {
+            this.amazonS3Client = AmazonS3ClientBuilder.standard()
+                    .withRegion(environment.readEnv(AWS_REGION))
```

Comment:
the environment variable can be read in directly when we you are creating the constant.